INSTALL.md

```diff
@@ -8,9 +8,9 @@ How to install application using helm:  <br />
     helm install ReleaseName ChartName
     
     a.) Add helm repository in tractusx:
-           helm repo add autosetup https://eclipse-tractusx.github.io/charts/tractusx-dev
+           helm repo add autosetup https://eclipse-tractusx.github.io/charts/dev
     b.) To search the specific repo in helm repositories 
-           helm search repo tractusx-dev
+           helm search repo autosetup/orchestrator
     c.) To install using helm command:   
-           helm install autosetup tractusx-dev/autosetup
+           helm install autosetup autosetup/orchestrator
```

Comment:
Getting error:
`Error: INSTALLATION FAILED: chart "orchestrator" matching  not found in autosetup index. (try 'helm repo update'): no chart name found`

Already proposed a solution.